modules/_labs/button/react/lib/Button.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from 'react';
+import styled from '@emotion/styled';
+import {ButtonVariant, ButtonSize} from './types';
+import {CanvasSystemIcon} from '@workday/design-assets-types';
+
+export interface ButtonProps extends React.HTMLAttributes<HTMLButtonElement> {
```

Comment:
I think we're all done with the `BaseButtonProps` + generics interfaces. It was super unclear what props were where. Hoping a slight bit of repetition makes things way clearer.